src/components/Activity/TimeTag.js

```diff
@@ -1,18 +1,22 @@
-import React from 'react'
-import { formatDistance } from 'date-fns'
+import React, { useEffect, useState } from 'react'
 import PropTypes from 'prop-types'
+import { formatDistance } from 'date-fns'
 
-export default function TimeTag({ style, date }) {
-  const [relativeTime, setRelativeTime] = React.useState('')
-  React.useEffect(() => {
+function TimeTag({ style, date }) {
+  const [relativeTime, setRelativeTime] = useState('')
+
+  useEffect(() => {
     const interval = setInterval(() => {
       const relativeTime = formatDistance(new Date(), new Date(date))
         .replace('about', '')
+        .replace('minutes', 'min')
         .trim()
```

Comment:
Could we use [`formatDistanceStrict()`](https://date-fns.org/v2.0.0-alpha.22/docs/formatDistanceStrict)?